src/tests/validity.rs

```diff
@@ -41,6 +41,20 @@ fn text_escaped() {
     test_amber!("echo \"Hello \\\"World\\\"\"", "Hello \"World\"");
 }
 
+#[test]
+fn text_isolated_from_bash() {
+    let code = "
+        unsafe $SUCCESS=\"FAILURE\"$
+        echo \"This is a $SUCCESS\"
+
+        echo \"Today is `date`\"
```

Comment:
@Mte90 Here I check for the `` ` `` symbol. If this string evaluates incorrectly then the `` `date` `` will evaluate to the literally current date. otherwise it'll be just what it is in the string literal.

Reply:
I don't think that this test should be split further into even smaller chunks. That will only increase the time to test the validity tests